capstone/src/main/java/com/google/univiz/api/SearchResource.java

```diff
@@ -18,5 +18,5 @@
    *     possible matches. The list is also not an exhaustive set of matches. It only includes few
    *     top results.
    */
-  List<SearchData> getSearchSuggestions(String collegeNames);
+  List<SearchData> getSearchSuggestions(String partialCollegeName);
```

Comment:
Since you made it static in the implementation you should make it static here too.